yarn-project/bb-prover/src/avm_proving_tests/avm_proving_tester.ts

```diff
@@ -0,0 +1,157 @@
+import { type AvmCircuitInputs, type AztecAddress, VerificationKeyData } from '@aztec/circuits.js';
+import { createLogger } from '@aztec/foundation/log';
+import { type PublicTxSimulationTester, type TestEnqueuedCall } from '@aztec/simulator/public/fixtures';
+
+import fs from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import path from 'path';
+
+import {
+  type BBResult,
+  type BBSuccess,
+  BB_RESULT,
+  generateAvmProof,
+  generateAvmProofV2,
+  verifyAvmProof,
+  verifyAvmProofV2,
+} from '../bb/execute.js';
+import { extractAvmVkData } from '../verification_key/verification_key_data.js';
+
+const BB_PATH = path.resolve('../../barretenberg/cpp/build/bin/bb');
+
+export class AvmProvingTester {
```

Comment:
Just noting that this is the only one not inheriting from other tester. Don't mind but noting it.